routes/web.php

```diff
@@ -89,3 +89,13 @@
  * Image Controller  *
  *-----------------------*/
 Route::get('image/device/{device_id}', 'ImageController@show');
+
+/*-----------------------*
+ * Location Controller *
+ *-----------------------*/
+Route::resource('location', 'LocationController');
```

Comment:
Add location.index and site.index to manage menu in layouts/dashboard and layouts/home as Manage Locations and Manage Sites

Reply:
Create a ON DELETE CASCADE foreign key relationship between sites and locations. So when a site is deleted so are it's locations. Something like this in a new migration:
```php
		Schema::table('locations', function(Blueprint $table) {
			$table->foreign('site_id')->references('id')->on('sites')
						->onDelete('cascade');
		});
```

Reply:
@brandonjank Added location.index and site.index to the home layout manage drop down list. They were already added to the dashboard layout. Also the delete cascade for when a site is deleted its connected locations also delete is already implemented.